localstack-core/localstack/aws/connect.py

```diff
@@ -26,6 +37,7 @@
     INTERNAL_AWS_SECRET_ACCESS_KEY,
     MAX_POOL_CONNECTIONS,
 )
+from localstack.dns.server import get_fallback_dns_server
```

Comment:
This import is causing the CLI tests to fail, I think :eyes: 

https://github.com/localstack/localstack-ext/actions/runs/11251001003/job/31281083714?pr=3517

Reply:
NVM, I just saw @dfangl's to fix it: https://github.com/localstack/localstack/pull/11658 :raised_hands: